<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="3" passed="2" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2024-03-03T20:31:31 IST" name="Default suite" finished-at="2024-03-03T20:31:48 IST" duration-ms="16660">
    <groups>
    </groups>
    <test started-at="2024-03-03T20:31:31 IST" name="Default test" finished-at="2024-03-03T20:31:48 IST" duration-ms="16660">
      <class name="Reports.LearnExtentReports">
        <test-method is-config="true" signature="generateReports()[pri:0, instance:Reports.LearnExtentReports@1972e513]" started-at="2024-03-03T20:31:31 IST" name="generateReports" finished-at="2024-03-03T20:31:32 IST" duration-ms="563" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- generateReports -->
        <test-method signature="myFirsttest()[pri:0, instance:Reports.LearnExtentReports@1972e513]" started-at="2024-03-03T20:31:32 IST" name="myFirsttest" finished-at="2024-03-03T20:31:44 IST" duration-ms="11790" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- myFirsttest -->
        <test-method signature="mysecondTest()[pri:0, instance:Reports.LearnExtentReports@1972e513]" started-at="2024-03-03T20:31:44 IST" name="mysecondTest" finished-at="2024-03-03T20:31:48 IST" duration-ms="4082" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- mysecondTest -->
        <test-method signature="thirdTest()[pri:0, instance:Reports.LearnExtentReports@1972e513]" started-at="2024-03-03T20:31:48 IST" name="thirdTest" finished-at="2024-03-03T20:31:48 IST" duration-ms="91" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[null]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: null
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.fail(Assert.java:116)
at Reports.LearnExtentReports.thirdTest(LearnExtentReports.java:62)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:228)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:961)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:201)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at org.testng.TestRunner.privateRun(TestRunner.java:819)
at org.testng.TestRunner.run(TestRunner.java:619)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:443)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)
at org.testng.SuiteRunner.run(SuiteRunner.java:336)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1228)
at org.testng.TestNG.runSuites(TestNG.java:1134)
at org.testng.TestNG.run(TestNG.java:1101)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- thirdTest -->
      </class> <!-- Reports.LearnExtentReports -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
